src/components/NcRichText/autolink.js

```diff
@@ -23,10 +23,10 @@ const NcLink = {
 	},
 }
 
-export const remarkAutolink = function({ autolink, useMarkdown }) {
+export const remarkAutolink = function({ autolink, useMarkdown, useExtendedMarkdown }) {
 	return function(tree) {
-
-		if (!useMarkdown || !autolink) {
+		// remark-gfm has its own autolink parser
+		if (useExtendedMarkdown || !useMarkdown || !autolink) {
```

Comment:
Does it mean that with extended MD auto-link is always enabled even when autoLink === false?

Reply:
Yes, there is no way to disable it